src/main/java/com/devonfw/quarkus/general/service/exception/mapper/AbstractExceptionMapper.java

```diff
@@ -0,0 +1,58 @@
+package com.devonfw.quarkus.general.service.exception.mapper;
+
+import java.time.ZonedDateTime;
+import java.util.HashMap;
+import java.util.Map;
+
+import javax.ws.rs.core.Context;
+import javax.ws.rs.core.MediaType;
+import javax.ws.rs.core.Response;
+import javax.ws.rs.core.UriInfo;
+
+import lombok.extern.slf4j.Slf4j;
+
+@Slf4j
+public abstract class AbstractExceptionMapper {
+
+  protected boolean exposeInternalErrorDetails = false;
+
+  @Context
+  UriInfo uriInfo;
+
+  protected void logError(Exception exception) {
+
+    log.error("Exception:{},URL:{},ERROR:{}", exception.getClass().getCanonicalName(), this.uriInfo.getRequestUri(),
+        exception.getMessage());
+  }
+
+  protected Response createResponse(int status, String errorCode, Exception exception) {
+
+    Map<String, Object> jsonMap = new HashMap<>();
+    jsonMap.put("errorCode", errorCode);
+    if (this.exposeInternalErrorDetails) {
+      jsonMap.put("message", getExposedErrorDetails(exception));
+    } else {
+      jsonMap.put("message", exception.getMessage());
+    }
+    jsonMap.put("uri", this.uriInfo.getPath());
+    jsonMap.put("timestamp", ZonedDateTime.now().toString());
+    return Response.status(status).type(MediaType.APPLICATION_JSON).entity(jsonMap).build();
+  }
```

Comment:
Was there a design decision to change our "contract"?
https://github.com/devonfw/devon4j/blob/master/documentation/guide-rest.asciidoc#rest-exception-handling

I am happy to discuss if we want to strip out UUID if we think we rather have OpenTelemetry now but it is a valid and established concept allowing display UUID to end-user and find stacktrace in logs in seconds if UUID is provded with support-ticket by end-user.

Reply:
No, I just missed it. I have added it now.